src/main.js

```diff
@@ -124,6 +128,7 @@ export async function rebuildNativeModules(nodeVersion, nodeModulesPath, whichMo
     `--target=${nodeVersion}`,
     `--arch=${arch || process.arch}`
   );
+  if (process.env.DEBUG) args.push('--verbose');
```

Comment:
I might be wrong here, but the `DEBUG` variable is normally set to a glob string that matches package names that you want to debug.  So just because `DEBUG` is not undefined does not mean the user wants to debug this specific module.